velox/common/base/BloomFilter.h

```diff
@@ -31,9 +34,27 @@ namespace facebook::velox {
 // expected entry, we get ~2% false positives. 'hashInput' determines
 // if the value added or checked needs to be hashed. If this is false,
 // we assume that the input is already a 64 bit hash number.
-template <bool hashInput = true>
+// case:
+// InputType can be one of folly hasher support type when hashInput = false.
+// InputType can only be uint64_t when hashInput = true.
+template <class InputType = uint64_t, bool hashInput = true>
 class BloomFilter {
  public:
+  explicit BloomFilter(HashStringAllocator* allocator)
+      : bits_{StlAllocator<uint64_t>(allocator)} {}
+
+  BloomFilter(const char* serialized, HashStringAllocator* allocator)
+      : bits_{StlAllocator<uint64_t>(allocator)} {
+    common::InputByteStream stream(serialized);
+    auto size = stream.read<int32_t>();
+    bits_.resize(size);
+    auto bitsdata =
+        reinterpret_cast<const uint64_t*>(serialized + stream.offset());
+    for (auto i = 0; i < size; i++) {
+      bits_[i] = bitsdata[i];
```

Comment:
`memcpy`?

Reply:
Now it just is used in test

Reply:
Then you can probably just create an empty one and use `merge`, and remove this constructor